dockstore-integration-testing/src/test/java/io/dockstore/client/cli/WorkflowIT.java

```diff
@@ -1048,9 +1048,21 @@ public void testManualRegisterThenPublish() throws ApiException {
         assertEquals("There should be two workflows with name altname, there are " + count2, 2, count2);
 
         // Publish github workflow
-        workflowApi.refresh(githubWorkflow.getId(), false);
+        Workflow refreshedWorkflow = workflowApi.refresh(githubWorkflow.getId(), false);
+        workflowApi.publish(githubWorkflow.getId(), publishRequest);
+
+        Optional<WorkflowVersion> testWDL = refreshedWorkflow.getWorkflowVersions().stream().filter(workflowVersion -> workflowVersion.getName().equals("testWDL")).findFirst();
```

Comment:
make a new branch/ or tag from `testWDL` with a descriptive name
This repo and branch is used in a bunch of places and it would be easy for someone to add a `meta` description without knowing that this test relied on not having one.